doc/source/whatsnew/v0.24.0.rst

```diff
@@ -8,6 +8,11 @@ v0.24.0 (Month XX, 2018)
    Starting January 1, 2019, pandas feature releases will support Python 3 only.
```

Comment:
Header format of v.0.24 not changed here? Then maybe the v.0.23-ones should stay in line with the style before/after v.0.23.

Reply:
If the new style starting from `0.23` should be: 
```diff
- v0.23.0 (May 15, 2018)
- ----------------------
+ What's new in 0.23.0 (May 15, 2018)
+ -----------------------------------
```

then this should be continued with v.0.24 (or reverted for the 0.23.* ones).